js/scripts.js

```diff
@@ -1,8 +1,8 @@
-AWS.config.update({
+/*AWS.config.update({
     region: 'eu-west-1',
     accessKeyId: 'fakeMyKeyId', // add this as env var
     secretAccessKey: 'fakeSecretAccessKey' // add this as env var
-});
+});*/
 
 var docClient = new AWS.DynamoDB.DocumentClient();
```

Comment:
I think this DynamoDB client is no longer needed too

Reply:
Resolved.